model/simple/taskbehavior.go

```diff
@@ -144,6 +144,7 @@ func (tb *TaskBehavior) Done(ctx model.TaskContext) (notifyFlow bool, taskEntrie
 			logger.Debugf("Task '%s' has %d outgoing links", ctx.Task().ID(), numLinks)
 		}
 
+		var hasFollow, hasOtherwise bool
```

Comment:
I would change hasFollow to linkFollowed
Also, I would just have an 'otherwiseLink' object that you assign the link to, that way it can serve as both your check if it has an otherwise link and also avoids you having to loop through the set of links again

Reply:
THanks, @fm-tibco  Updated code